src/components/Card/Card.js

```diff
@@ -1,11 +1,26 @@
 import './Card.css';
+import { processText } from '../../utils/helpers';
 
 function Card(props) {
+  const { processedText, startsWithEmphasis } = processText(props.card.text);
   return (
-    <li className="card">
-      <img className="card__image" src={props.card.image} alt="#" />
-      <h3 className="card__title">{props.card.title}</h3>
-      <p className="card__text">{props.card.text}</p>
+    <li className={`card card_type_${props.type}`}>
+      <div
+        className={`card__image card__image_type_${props.type}`}
+        style={{ backgroundImage: `url(${props.card.image})` }}
+      />
+      <h3 className={`card__title card__title_type_${props.type}`}>{props.card.title}</h3>
+      {props.card.job && <p className="card__job">{props.card.job}</p>}
+      <p className={`card__text card__text_type_${props.type}`}>
+        {processedText.map((result) => (
+          <>
+            <span key={Math.random()} className={`card__text ${startsWithEmphasis ? 'card__text_emphatic' : ''}`}>
+              {result[0]}
+            </span>
+            <span key={Math.random()} className={`card__text ${!startsWithEmphasis ? 'card__text_emphatic' : ''}`}>{result[1]}</span>
```

Comment:
I'm curious, what is result[1]/result[0]?

Reply:
`processedText` returns an array of arrays, like

```
[ [ "some text", "some more text" ], [ ... ] , ... ]
```

So each `result` is one of those subarrays:  `[ "some text", "some other text" ]`.